src/EventStore.ClientAPI/ExpectedVersion.cs

```diff
@@ -29,7 +29,6 @@ public static class ExpectedVersion {
 		/// <summary>
 		/// The stream should exist but be empty when writing. If it does not exist or is not empty treat that as a concurrency problem.
 		/// </summary>
-		[Obsolete("ExpectedVersion.EmptyStream has been deprecated. Use ExpectedVersion.NoStream instead")]
```

Comment:
Why the Obsolete decorator is being remove?

Reply:
It was mistake. Fixed! Thanks! :)